app/src/main/java/com/hajduk/systems/prepareordermailing/holder/ApplicationDataHolder.kt

```diff
@@ -0,0 +1,75 @@
+package com.hajduk.systems.prepareordermailing.holder
+
+import com.hajduk.systems.prepareordermailing.adapter.woocommerce.model.CustomerDto
+import com.hajduk.systems.prepareordermailing.adapter.woocommerce.model.OrderDto
+import java.io.File
+
+class ApplicationDataHolder {
```

Comment:
@ihajduk Może nie jest to najpiękniejsza klasa wszechczasów, ale to chyba najlepsze rozwiązanie jeśli chodzi o przekazywanie bardziej skomplikowanych struktur danych między widokami